src/Router.js

```diff
@@ -11,6 +12,7 @@ const Router = () => {
         <Route path="/" element={<Main />} />
         <Route path="/productlist" element={<ProductList />} />
         <Route path="/login" element={<LoginModal />} />
+        <Route path="/nav" element={<Nav />} />
```

Comment:
Nav component는 모든 컴포넌트에서 출력이 되야 하므로 아래 코드처럼 되야 할 것 같습니다.
```suggestion
        <Nav /> 
        <Route path="/" element={<Main />} />
        <Route path="/productlist" element={<ProductList />} />
        <Route path="/login" element={<LoginModal />} />
```